package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "react-native-simple-store",
-  "version": "1.3.0",
+  "version": "1.4.0",
```

Can you folks help me think this through?

Since it seems like the developer now needs to run `react-native link` in order to get this new version to work, would that make this a breaking change and require a `2.0.0` version?

I'd be interested to hear feedback since it's not super clear because the `react-native-simple-store` API hasn't changed but it does require additional setup.

---

Yeah, you're right. I didn't think of that. Should I change the version to 2.0?

```suggestion
  "version": "2.0.0",
```

---

Yeah, let's play it on the safe side and just bump it to `2.0.0`. That way it's more clear that additional action is required.